package.json

```diff
@@ -31,6 +31,10 @@
     "malevic": "0.18.2"
   },
   "devDependencies": {
+    "@rollup/plugin-commonjs": "^11.1.0",
```

Comment:
Let's use strict packages' versions (without `^`). This way we will ensure that everyone (including reviewers from Mozilla) is using the same packages, and has the same build results and bugs. Also this can protect from vulnerabilities like [this](https://www.theregister.co.uk/2016/03/23/npm_left_pad_chaos/) or [this](https://snyk.io/blog/malicious-code-found-in-npm-package-event-stream/).

Reply:
Woops seems like --save-dev isn't the best option for automatic Install smh

Reply:
You can add `--save-exact` or `-E`

Reply:
Noted